src/domain/messages/messages.repository.ts

```diff
@@ -46,16 +46,16 @@ export class MessagesRepository implements IMessagesRepository {
     message: unknown,
     safeAppId: number | null,
     signature: string,
-  ): Promise<Message> {
+  ): Promise<unknown> {
     const transactionService =
       await this.transactionApiManager.getTransactionApi(chainId);
-    const createdMessage = await transactionService.postMessage(
+
```

Comment:
Nit: can we add a TODO comment explaining why we've removed the validation, linking to the [transaction service issue](https://github.com/safe-global/safe-transaction-service/issues/1517)? We can then revisit this in the future.

Reply:
Yes, I agree with @fmrsabino but I appreciate your suggestion @iamacook 🙂 Also, I think having the Transaction Service documentation issue linked here helps to maintain the traceability just in case some doubt related to this appears in the future.